tcms/core/contrib/linkreference/views.py

```diff
@@ -90,8 +89,8 @@ def remove(request, link_id):
     try:
         value = field.clean(link_id)
     except ValidationError as err:
-        jd = json.dumps({'rc': 1, 'response': '\n'.join(err.messages)})
-        return http.HttpResponseBadRequest(content=jd, content_type='application/json')
+        json_data = json.dumps({'rc': 1, 'response': '\n'.join(err.messages)})
+        return http.HttpResponseBadRequest(content=json_data, content_type='application/json')
```

Comment:
`JsonResponse` with `status=http.client.BAD_REQUEST`

Reply:
Done. I also removed `json.dumps` on line 92, because `http.JsonRequest` does `json.dumps()` internally. The same thing I did for the other place where I replaces `http.HttpResponseBadRequest` with `http.JsonResponse`